src/corehost/build.proj

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I chose this way because currently core-setup was depending on build_project which involved using some bootsrapper that called cli and msbuild. I understand that needs to change, if not on this PR, I'll open an issue.